book/src/advanced_re-orgs.md

```diff
@@ -2,6 +2,10 @@
 
 Since v3.4.0 Lighthouse will opportunistically re-org late blocks when proposing.
 
+When Lighthouse is about to propose a new block, it quickly checks whether the block from the previous slot landed so late that hardly anyone attested to it.
+If that late block looks weak enough, Lighthouse may decide to “re-org” it away: instead of building on it, Lighthouse builds its new block on the grand-parent block, turning the late block into an orphan.
+Because Lighthouse only does this when the opportunity presents itself and the odds of success are good, the code calls the behaviour “opportunistically re-organising late blocks when proposing.
```

Comment:
I think the last sentence makes it too obvious that the paragraph above was written by a different author :sweat_smile: . I think we could could remove the "Since v3.4.0 Lighthouse will opportunistically re-org late blocks when proposing" paragraph altogether, and re-phrase (or remove) this sentence?

Reply:
I removed the last sentence.